.ci/common-qe/common-qe-configuration.conf

```diff
@@ -0,0 +1,21 @@
+# Environment
+env.root.dir.path=
+env.setup.environment.script.path=
+env.setup.environment.method.name=
+
+# Build
+build.script.path=cico_functions.sh
+build.and.push.method.name=build_and_push
```

Comment:
Why do we need to standardize/unify the "build" part of the process... This will/can be pretty different on each of the repos.
I think it would be OK if the scripts in each repo will handle the build part separately.

Reply:
Agree, that is why 
```
build.script.path=cico_functions.sh
build.and.push.method.name=build_and_push
```
contains a path to file which contain build command and name of this command for processing your described issue.
In other words, this is delegating to the dedicated repo build method.

Reply:
I mean if in some particular repo the build command located in the ```"repo/example.build_script.sh"```

and name of this method
```"example_build_script""```
you only need to reuse them by configuring
```
build.script.path=repo/example.build_script.sh
build.and.push.method.name=example_build_script
```

Reply:
And what's the benefit of this against having this in the repo:
```
# Build & push
. repo/example.build_script.sh
example_build_script
```
Overall I like the approach, but this one particular part seems like unecessary complication of the process (or I fail to see its benefit yet)

Reply:
Ok, I'll remove it